### DISCUSSION TITLE
Building Interface Extensions with .env

### DISCUSSION THREAD

**Discussion by HerrHase** [directus/directus#19192] [Feature Requests]
### Summary

I have tried to use environments variables to building a Interface Extensions.

I add a .env file to the main directory to my extension. But i was unable to use the variables. cross-env, set, i had add to the script in package.json, nothing worked, i also tried dotenv to the index.js. Maybe i doing something wrong. Have anyone figured this out?

### Motivation

-

### Detailed Design

-

### Requirements List

-

### Drawbacks

-

### Alternatives

-

### Adoption Strategy

-

**Comment by rijkvanzanten** [comment]
Closing this as it's not a feature request 👍🏻  For questions, our Discord server is a better place 🙂 https://directus.chat
